SlimApp/SignUp/SignUpView.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  SignUpView.swift
+//  SlimApp
+//
+//  Created by Ekaterina Grishina on 25/01/23.
+//
+
+import SwiftUI
+
+struct SignUpView: View {
+
+    @ObservedObject var viewModel: SignUpViewModel
+
+    var body: some View {
+        VStack {
+            Text("Create an account")
+                .font(.title)
+                .padding(.bottom, 64)
+            TextField("", text: $viewModel.email, prompt: Text("Email"))
```

Comment:
Maybe using the label without prompt is better in order to have less problems in the future with Dark Mode

Reply:
I've changed a bit colors for dark mode, so now I need to use prompt in order placeholder to be brighter in dark mode

Reply:
<img width="530" alt="image" src="https://user-images.githubusercontent.com/15235887/214864870-0a892518-440a-4c9d-810e-d822e0b5b7d7.png">